docs/references/concepts.md

```diff
@@ -4,16 +4,35 @@ description: Fundamental user-facing concepts
 
 # Key Concepts
 
+## Instance
+
+An "instance" of Opstrace runs inside your cloud account.
+It is installed with the CLI `opstrace create` command.
+This name must be _globally_ unique and limited to 23 characters (`[a-z0-9-_]`) because it used to provide a conveient DNS name:  `<instance_name>.opstrace.io`.
+Once chosen, an instance name cannot be changed.
+
 ## Tenants
 
-The concept of multitenancy allows for isolating environments within the same Opstrace cluster.
-Tenants are logically isolated, but otherwise share the same \(physical\) base system.
+A "tenant" is a unit of isolation.
+[Multi-tenant systems](https://en.wikipedia.org/wiki/Multitenancy) allow for logical separation of concerns, while otherwise sharing the same underlying system.
+
+Opstrace instances support multiple, secured tenants.
+The tenant primitive extended from the underlying projects—Cortex and Loki.
+Tenants isolate isolate data, dashboards, and API endpoints.
+For example, isolate by:
+
+* Teams (e.g., `team-revenue` vs. `team-fraud`)
+* Environment (e.g., `prod`, `staging`, `dev`)
+* A combination of these things, or anything else important to you.
```

Comment:
I think this is not a third example, and should therefore not be in the list, but below it as a new sentence.